src/styles/blocks/categories.scss

```diff
@@ -0,0 +1,71 @@
+.categories {
+  &__wrapper {
```

Comment:
Consider using a more descriptive class name for the wrapper element, like `categories__grid`.